src/sentry/static/sentry/app/views/organizationStats/organizationStatsDetails.jsx

```diff
@@ -138,4 +141,9 @@ class OrganizationStats extends React.Component {
   }
 }
 
+const StyledPageHeader = styled(PageHeader)`
+  margin-bottom: ${space(3)};
```

Comment:
Why aren't these just default values?

Reply:
You mean the default margin-bottom for an `h1`? It would be based on ems, rather than pixels, and wouldn't be consistent with our spacing system.

Reply:
oh nvm i see what you are saying. I can explain that too one sec

Reply:
I didn't include this because (as you probably know) packaging in external whitespace with a component can be really annoying and create a ton of overrides down the line.

Even internal whitespace can cause annoying extensibility issues when set as the default (see `withButtons` prop in `panelHeader`). But I can (and should) add a `withPadding` prop to at least reduce all this duplication.

Reply:
this change is up too